mypyc/irbuild/statement.py

```diff
@@ -258,29 +302,25 @@ def transform_import_from(builder: IRBuilder, node: ImportFrom) -> None:
         module_package = ""
 
     id = importlib.util.resolve_name("." * node.relative + node.id, module_package)
-
-    globals = builder.load_globals_dict()
-    imported_names = [name for name, _ in node.names]
-    module = builder.gen_import_from(id, globals, imported_names, node.line)
-
-    # Copy everything into our module's dict.
+    builder.imports[id] = None
```

Comment:
Similar to above, what about only using the table based option outside functions? Imports within functions are already something of a performance bottleneck, so it would be nice if we could keep them as fast as they are currently. (Eventually I'd like us to make nested imports faster, but that might take a while.)

Reply:
I can't measure any slowdown running `from typing import Any` in a tight loop with the table-driven code.[^1] In fact, the table-driven code can be faster if there are several names. (I tried six.)

```
**one name**

[new] mean: 11.7 (stdev: 0.273) | min 11.4 | max: 12.2
[new] mean: 12.0 (stdev: 0.530) | min 11.3 | max: 13.0
[old] mean: 12.8 (stdev: 0.492) | min 11.8 | max: 13.5
[old] mean: 12.9 (stdev: 0.606) | min 12.0 | max: 13.8

**six names**

[new] mean: 15.9 (stdev: 0.543) | min 14.9 | max: 16.7
[new] mean: 15.0 (stdev: 0.365) | min 14.6 | max: 15.6
[old] mean: 18.8 (stdev: 0.358) | min 18.3 | max: 19.3
[old] mean: 18.4 (stdev: 0.451) | min 17.6 | max: 19.1
```

Since the table-driven code doesn't seem to introduce any slowdown, I won't bring back the old `from import` code.

[^1]: My data suggests it's actually faster, but that seems suspect. I won't call it any faster as the speedup is within the margin of error.